packages/app/src/app/pages/Dashboard/Sidebar/index.tsx

```diff
@@ -743,8 +743,25 @@ const NestableRowItem: React.FC<NestableRowItemProps> = ({
             <Element as="span" css={css({ width: 4, flexShrink: 0 })} />
           )}
 
-          <Stack align="center" gap={2} css={{ width: 'calc(100% - 28px)' }}>
-            <Icon name="folder" size={24} css={css({ flexShrink: 0 })} />
+          <Stack
+            align="center"
+            gap={2}
+            css={{
+              paddingLeft: 0,
+              paddingRight: 0,
+              marginRight: '0!important',
+            }}
+          >
+            <Stack
+              justify="center"
+              align="center"
+              css={{
+                width: 24,
+                marginRight: '8px!important',
```

Comment:
When I remove the `!important` I don't see an immediate difference, do we need it? It might be related to something I can't immediately see.